compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -581,6 +581,16 @@ impl CStore {
     ) -> Span {
         self.get_crate_data(cnum).get_proc_macro_quoted_span(id, sess)
     }
+
+    pub fn get_ambiguity_module_children_untracked(
```

Comment:
```suggestion
    pub fn ambiguity_module_children_untracked(
```
Nit: naming conventions.